src/components/query/PropertyQueryEditor.tsx

```diff
@@ -77,8 +77,9 @@ export class PropertyQueryEditor extends PureComponent<Props, State> {
     const { query } = this.props;
     const assetChanged = query?.assetId !== oldProps?.query?.assetId;
     const propChanged = query?.propertyId !== oldProps?.query?.propertyId;
-    if (assetChanged || propChanged) {
-      if (!query.assetId) {
+    const regionChanged = query?.region !== oldProps?.query?.region;
+    if (assetChanged || propChanged || regionChanged) {
```

Comment:
Is this the one real change in the the PR?

Reply:
Yes - just forcing an updating in that hook